### PR TITLE
Replace httparty with net/http

### PR DESCRIPTION
**Pull request by RKushnir**
http://www.mikeperham.com/2016/02/09/kill-your-dependencies/

### CODE REVIEW

File: lib/soapy_cake/client.rb [hunk 1]
```diff
@@ -63,9 +65,22 @@ def response_body(request)
     def http_response(request)
       logger.info("soapy_cake:request #{request}") if logger
 
-      url = "https://#{domain}#{request.path}"
-      HTTParty.post(url, headers: HEADERS, body: request.xml, timeout: NET_TIMEOUT).tap do |res|
-        raise RequestFailed, "Request failed with HTTP #{res.code}: #{res.body}" unless res.success?
+      uri = URI::HTTPS.build(host: domain, path: request.path)
```
Comment (dziemba) [inline lib/soapy_cake/client.rb]: Do we need to build an URI object at all? `Net::HTTP::Post.new` (also) takes a path as its first argument.

Comment (RKushnir) [inline lib/soapy_cake/client.rb]: Right, removed.

**Comment by dziemba** [comment]
Nice, I like the idea!

🍄

**Comment by tsujigiri** [comment]
:v:

**Pull request merged**